src/components/Causes/css.js

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import EventPhoto from '../../images/womenintech-6.jpg';
+
+const CausesSection = styled.section`
+	@media (min-width: 850px) {
+		background-color: #ffffff;
+	}
+
+	@media (max-width: 849px) {
+		background-image: url(${EventPhoto});
+		background-position: center center;
+		background-size: cover;
+	}
+`;
+
+const CausesPhoto = styled.section`
+	@media (min-width: 850px) {
+		background-image: url(${EventPhoto});
+		background-size: cover;
+		position: relative;
+		height: 500px;
+		width: 800px;
+		left: 5rem;
+		top: 6rem;
+	}
+`;
+
+const CausesTextSection = styled.section`
+	@media (min-width: 1025px) {
+		background-color: #ffffff;
+		position: relative;
+		height: 450px;
+		width: 700px;
+		margin-left: 32rem;
+		top: -11rem;
+		margin-top: -12.5rem;
```

Comment:
Typically we want to avoid negative margin as it distorts the box model. If you really need to move something by a small amount I would use transform: translate instead. This looks like a big number though. Are you sure there's not another way you could deal with it by using padding?